memory/src/main/scala/filodb.memory/format/vectors/Histogram.scala

```diff
@@ -324,6 +327,91 @@ final case class MaxHistogram(innerHist: MutableHistogram, max: Double) extends
   }
 }
 
+final case class DDSHistogram(relativeAccuracy: Double) extends HistogramWithBuckets {
+  def serialize(intoBuf: Option[MutableDirectBuffer] = None): MutableDirectBuffer = ???
+
+  val logGamma: Double = Math.log((1 + relativeAccuracy) / (1 - relativeAccuracy))
+  val maxIndexedValue = maxIndexableValue
+  val minIndexedValue = minIndexableValue
+  var zeroCount = 0
+  var indexMapping = new CubicallyInterpolatedMapping(relativeAccuracy)
+  var store = new UnboundedSizeDenseStore
+  final def bucketValue(no: Int): Double = 0
+  def buckets: HistogramBuckets = ???
+
+  def getCount: Long = zeroCount + store.getTotalCount
+
+  override def quantile(q: Double): Double = {
+    getValueAtQuantile(q, getCount)
+  }
+
+  def getValueAtQuantile(quantile: Double, count: Long): Double = {
```

Comment:
I suppose you had to implement this due to some reason you could not use the underlying DDSketch's own code?  Please add comment